tests/pageObjects/AddPage.js

```diff
@@ -0,0 +1,41 @@
+class AddPage{
+  constructor() 
+  {
+    this.accountButtonSelector = "//button[contains(@id,'account-button')]";
+    // this.accountButtonSelector = "button#account-button";
+    this.adminPanelSelector = "//div[contains(@id,'link-to-account')]//a[contains(@href,'/admin/orders')]";
```

Comment:
Can you make this selector shorter?
```suggestion
    this.adminPanelSelector = "//div[contains(@id,'link-to-account')]//a[contains(@href,'/admin/orders')]";
```